src/etc/inc/unbound.inc

```diff
@@ -156,14 +167,26 @@ function unbound_generate_config_text($unboundcfg = NULL, $cfgsubdir = "") {
 	// Setup optimization
 	$optimization = unbound_optimization();
 
+	$module_config = '';
+
+	// Setup Python module (pre validator)
+	if (isset($unboundcfg['python']) && !empty($unboundcfg['python_script']) && $unboundcfg['python_order'] == 'pre_validator') {
+		$module_config .= 'python ';
+	}
+
 	// Setup DNSSEC support
 	if (isset($unboundcfg['dnssec'])) {
-		$module_config = "validator iterator";
+		$module_config .= 'validator ';
```

Comment:
why don't you double quotes this string ? as it was...

any string shoud be double quoted i think...
you should check all over your code and change it
even if i know, it is not wrong syntaxically

Reply:
https://stackoverflow.com/questions/3446216/what-is-the-difference-between-single-quoted-and-double-quoted-strings-in-php

Reply:
i know all of that, 
it is just a question of cosmetic and to respect the existing code convention.

Reply:
Its a style preference.... pfSense older code is mostly double quotes, and most newer code is single quotes. Lets see what the devs say and can change if they wish.

Reply:
Single quotes are safer, unless you have a need to substitute variables in the string, it's better to err on the side of caution and use single quotes. We don't have any stated policy in the developer guide about which to use, so there is no need to follow any kind of tradition here.